modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,195 @@
+// Module included in the following assemblies:
+// * observability/network_observability/netobserv-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-reference_{context}"]
+= oc netobserv CLI reference 
+The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+
+.`oc netobserv` syntax
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
+----
+<1> Feature options can only be used with the `oc netobserv flows` command. They cannot be used with the `oc netobserv packets` command.
+
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Command| Description
+
+| `flows`
+| Capture flows information. For subcommands, see the "Flow capture subcommands" table. 
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as `netobserv packets --filter=tcp,80`. For more information about packet capture, see the "Packet capture subcommand" table.
+
+| `cleanup`
+| Remove the Network Observability CLI components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+[id="cli-reference-feature-commands"]
+== Network Observability Operator feature options
+The Network Observability Operator features can only be enabled when capturing flows. They do not work with packet capture.
```

Comment:
I think we should either clarify or get rid of that mention.

Here is my clarification suggestion:
```suggestion
[id="cli-reference-feature-commands"]
== Network Observability enrichment
The Network Observability enrichment to display zone, node, owner and resource names including optionnal features about packet drop, DNS latencies and round trip time can only be enabled when capturing flows. These will not appear in packet capture pcap output file.
```
Feel free to adapt

Reply:
I like this actually a lot better than invoking the Operator here. Good idea, thanks Julien.